.github/workflows/argilla-sdk.docs.yml

```diff
@@ -7,28 +7,16 @@ concurrency:
 on:
   push:
     paths:
-      - "argilla-sdk/**"
+      - "argilla-sdk/docs/**"
```

Comment:
```suggestion
      - "argilla-sdk/**"
```
I think we also need to take into account general code changes to include the potential API reference update?

Reply:
So, could we just add `argilla-sdk/src/**`?

Reply:
yes.